### PR TITLE
[sf-lodash] time format changed to support 1 digit hour

### PR DESCRIPTION
**Pull request by farhadbahrehmandhenry**

### CODE REVIEW

File: src/label-for-moment/label-for-moment.js [hunk 1]
```diff
@@ -1,17 +1,16 @@
 import _ from 'lodash';
 import moment from 'moment';
```
Comment (maxhudson) [inline src/label-for-moment/label-for-moment.js:2]: please remove - this shouldn't be here

File: src/label-for-moment/label-for-moment.test.js [hunk 2]
```diff
@@ -3,7 +3,21 @@ import moment from 'moment';
 import _ from 'lodash';
 
 describe('_.labelForMoment()', () => {
-  it('should return in 00:00am format if the moment is in today', () => {
+  it('should return the time in the right format', () => {
```
Comment (maxhudson) [inline src/label-for-moment/label-for-moment.test.js:6]: the previous it description was more specific/clear

Comment (farhadbahrehmandhenry) [reply src/label-for-moment/label-for-moment.test.js:6]: this is my new test, I still have the previous one.

Comment (maxhudson) [reply src/label-for-moment/label-for-moment.test.js:6]: ok - I merged it, but generally best not to have extra tests if you don't need them - it's unclear from both the descriptor in the it string and from the expectations what you're testing here that's new

Comment (farhadbahrehmandhenry) [reply src/label-for-moment/label-for-moment.test.js:6]: makes sense. thanks

**Pull request merged**